app/views/users/admin.html.erb

```diff
@@ -1,3 +1,16 @@
 <h1>Admin Tasks Page</h1>
+<h2>
+  <%= link_to "Menu Page", root_path) %>
+</h2>
+<%= link_to "Menu Page" root_path) %>
```

Comment:
Syntax error: you have an ending paren and no starting paren. Also, no comma after the link text in the second link. Also also, why are there 2 links?